sql/moz-fx-data-shared-prod/adjust_derived/adjust_derived_v1/query.py

```diff
@@ -0,0 +1,305 @@
+                                                }
+                                                fields_list.append(field_dict)
+
+    return fields_list
+
+
+def upload_to_bigquery(csv_data, project, dataset, adjust_list_part, date):
+    """Upload the data to bigquery."""
+    print(f"writing json to csv for {adjust_list_part}")
+
+    partition = f"{date}".replace("-", "")
+
+    with tempfile.NamedTemporaryFile() as tmp_csv:
+        with open(tmp_csv.name, "w+b") as f_csv:
+            write_dict_to_csv(csv_data, f_csv.name)
+
+            client = bigquery.Client(project)
+
+            print("hey, I'm about to config the job\n")
+
+            job_config = bigquery.LoadJobConfig(
+                create_disposition="CREATE_IF_NEEDED",
+                write_disposition="WRITE_TRUNCATE",
+                time_partitioning=bigquery.TimePartitioning(
+                    type_=bigquery.TimePartitioningType.DAY,
+                    field="date",
+                ),
+                skip_leading_rows=1,
+                schema=[
+                    bigquery.SchemaField("date", "DATE"),
+                    bigquery.SchemaField("app", "STRING"),
+                    bigquery.SchemaField("app_token", "STRING"),
+                    bigquery.SchemaField("network", "STRING"),
+                    bigquery.SchemaField("network_token", "STRING"),
+                    bigquery.SchemaField("campaign", "STRING"),
+                    bigquery.SchemaField("campaign_token", "STRING"),
+                    bigquery.SchemaField("adgroup", "STRING"),
+                    bigquery.SchemaField("adgroup_token", "STRING"),
+                    bigquery.SchemaField("creative", "STRING"),
+                    bigquery.SchemaField("creative_token", "STRING"),
+                    bigquery.SchemaField("country", "STRING"),
+                    bigquery.SchemaField("os", "STRING"),
+                    bigquery.SchemaField("device", "STRING"),
+                    bigquery.SchemaField("clicks", "INTEGER"),
+                    bigquery.SchemaField("installs", "INTEGER"),
+                    bigquery.SchemaField("limit_ad_tracking_install_rate", "FLOAT"),
+                    bigquery.SchemaField("click_conversion_rate", "FLOAT"),
+                    bigquery.SchemaField("impression_conversion_rate", "FLOAT"),
+                    bigquery.SchemaField("sessions", "INTEGER"),
+                    bigquery.SchemaField("daus", "INTEGER"),
+                    bigquery.SchemaField("waus", "INTEGER"),
+                    bigquery.SchemaField("maus", "INTEGER"),
+                ],
+            )
+
+            # Table names are based on the app name seen in the Adjust dashboard"
+            destination = f"{project}.{dataset}.adjust_deliverables_v1_${partition}"
+            print(destination)
+
+            job = client.load_table_from_file(f_csv, destination, job_config=job_config)
+
+            print(f"Running job {job.job_id}")
```

Comment:
Perhaps merging line 252 and this one?
```suggestion
            print(f"Writing adjust data for all apps to {destination}. BigQuery job ID: {job.job_id}")
```